test/python/test_user_config.py

```diff
@@ -37,6 +37,7 @@ def test_invalid_circuit_drawer(self):
         test_config = """
         [default]
         circuit_drawer = MSPaint
+        circuit_mpl_style = default
```

Comment:
Did you want to add any dedicated tests to validate the behavior of invalid options for `circuit_mpl_style`?

Reply:
I am not really worried about it, but I will add one.